librz/bin/p/bin_avr.c

```diff
@@ -88,11 +86,13 @@ static bool check_buffer_rjmp(RzBuffer *b) {
 	if (dst < 1 || dst > rz_buf_size(b)) {
 		return false;
 	}
-	tmp_entry = dst;
+	if (entry) {
+		*entry = dst;
+	}
 	return true;
 }
 
-static bool check_buffer_jmp(RzBuffer *b) {
+static bool check_buffer_jmp(RzBuffer *b, ut64 *entry) {
```

Comment:
```suggestion
static bool check_buffer_jmp(RzBuffer *b, RZ_OUT ut64 *entry) {
```

It is probably better to mark it in this case. Also below.